server/src/main/java/org/opensearch/cluster/routing/allocation/ExistingShardsAllocator.java

```diff
@@ -60,6 +60,13 @@ public interface ExistingShardsAllocator {
         Setting.Property.PrivateIndex
     );
 
+    public static final Setting<Boolean> EXISTING_SHARDS_ALLOCATOR_BATCH_MODE_ENABLED = Setting.boolSetting(
```

Comment:
nit: Rename to `EXISTING_SHARDS_ALLOCATOR_BATCH_MODE` ?

Reply:
done